cookbooks/aws-parallelcluster-config/files/ubuntu/network_interfaces/configure_nw_interface.sh

```diff
@@ -47,6 +47,7 @@ network:
   ethernets:
     ${DEVICE_NAME}:
 $STATIC_IP_CONFIG
+      mtu: '9001'
```

Comment:
I'd extend the commit message adding more information about this change. What we were missing and why we're introducing it (with a link to MTU best practices).

Reply:
Done